api/src/pcapi/models/feature.py

```diff
@@ -56,6 +56,7 @@ class FeatureToggle(enum.Enum):
     ENABLE_NATIVE_ID_CHECK_VERBOSE_DEBUGGING = (
         "Active le mode debug Firebase pour l'Id Check intégrée à l'application native"
     )
+    ENABLE_NEW_BOOKING_FILTERS = "Permet de filtrer les résarvations via de nouveaux filtres sur les statuts (réservation/validation/remboursement)"
```

Comment:
Est-ce que l'on a moyen de faire un peu moins verbeux ?
```suggestion
    ENABLE_NEW_BOOKING_FILTERS = "Active les nouveaux filtres sur les statuts pour la page de réservations"
```

Reply:
En effet ce sera plus limpide comme ca